final_prep/mod_zero_hero.rb

```diff
@@ -55,18 +80,59 @@
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
+scary_monster = {
+  name: "Blarg",
+  smell: "Copper and Gardenia",
+  weight: 312,
+  citiesDestroyed: ["Carmel, IN", "Elizabeth Township, NJ"],
+  luckyNumbers: [7, 13, 21],
+  address: {
+    number: 515,
+    street: "Walla Walla Way",
+    state: "Washington",
+    zip: 12345
+  }
+}
 
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
-
+#   - energy_level = 50
+class SuperHero
+  @@arch_nemesis = "The Syntax Error"
```

Comment:
I'd advise making these instance variables - with only 1 `@` and inside the initialize method.